packages/pangea-sdk/pangea/services/authn/authn.py

```diff
@@ -158,8 +152,6 @@ def __init__(
             self.password = AuthN.Client.Password(token, config, logger_name=logger_name)
             self.token_endpoints = AuthN.Client.Token(token, config, logger_name=logger_name)
 
-        # https://pangea.cloud/docs/api/authn#get-user-client-token
-        # - path: authn::/v1/client/userinfo
         def userinfo(self, code: str) -> PangeaResponse[m.ClientUserinfoResult]:
```

Comment:
@TimNewshamPangea @pangea-andrest @ggallien13 Looking at the latest JSON schema `v2_client_userinfo.json`, it looks like this got moved under user management now? Is this correct?

```
 "tags": ["user management"],
```

Should this still be within the `Client`, or should it be moved to the `User` class?